index.html

```diff
@@ -998,16 +998,90 @@
 
 </div>
 
-## Safety Systems {#safety}
+## Safety, Abuse, and Fraud {#safety}
+
+There are some cases where privacy and safety are in tension. These cases are more rare
```

Comment:
Concern: These examples under represent the harms and difficulties privacy induces on public safety work. Here are some further examples:

- Privacy of a dishonest publisher facilitates distribution of false information, malware, fraud
- Privacy of a dishonest adult, impersonating a child, enables them to solicit children on a major social media platform, vectoring them into a E2EE environment where criminal acts can occur with military-grade secrecy.
- Privacy-related concerns at major online service providers results in identity services that fail to support users who want to use transparent identity online as a part of legitimate professional activity. It makes it difficult or impossible for users seeking services to differentiate honest professionals from others who are indifferent or hostile to honest identity. The whole of the open Web is harmed.
- Privacy-centric identity systems enable ambiguous identity services, facilitating impersonation of specific individuals (e.g. oprah) and of classes of individuals (americans, BLM activists) facilitating in many forms of fraud and deception.
- PEW research (2014) has reported that about half of online harassment benefits from and is to some degree enabled by anonymity. In other words, weak identity up-to doubles the harassment experienced online. Such harassment disproportionately targets individuals in vulnerable subgroups. 
https://www.pewresearch.org/internet/2014/10/22/online-harassment/

Reply:
These are valid and also categorically distinct from Jeffrey's list of "tensions," in that they are scenarios in which privacy is weaponized / used to perpetrate harm. Could this accompany Jeffrey's list (e.g. "in addition to these tensions, there are also cases in which privacy becomes a harbor for abuse:")?